### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Dark-CodeX**
details about graphing calculator in README.md file

### CODE REVIEW

**Comment by Dark-CodeX** [comment]
Please merge my PR to main branch.

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Calculator ships regularly with new features and bug fixes. You can get the late
 ## Features
 - Standard Calculator functionality which offers basic operations and evaluates commands immediately as they are entered.
 - Scientific Calculator functionality which offers expanded operations and evaluates commands using order of operations.
+- Graphing Calculator functionality which offers fast and precise graphs to almost every mathematical equations in 2-D.
```
Comment (mcooley) [inline README.md:14]: I'd prefer that we say something shorter and simpler: `- Graphing Calculator for visualizing math equations.`

Comment (Dark-CodeX) [reply README.md:14]: I think mine is better and provide more info about `graphing calculator`

**Comment by mcooley** [comment]
Thanks for your interest in contributing. Adding a mention of graphing calculator in the README is a good idea, but I don't want to take this change as-is.

**Pull request closed without merging**